packages/flutter_markdown/lib/src/builder.dart

```diff
@@ -385,16 +385,31 @@ class MarkdownBuilder implements md.NodeVisitor {
       final _BlockElement current = _blocks.removeLast();
       Widget child;
 
-      if (current.children.isNotEmpty) {
-        child = Column(
-          mainAxisSize: MainAxisSize.min,
-          crossAxisAlignment: fitContent
-              ? CrossAxisAlignment.start
-              : CrossAxisAlignment.stretch,
-          children: current.children,
+      Widget defaultChild() {
+        if (current.children.isNotEmpty) {
+          return Column(
+            mainAxisSize: MainAxisSize.min,
+            crossAxisAlignment: fitContent
+                ? CrossAxisAlignment.start
+                : CrossAxisAlignment.stretch,
+            children: current.children,
+          );
+        } else {
+          return const SizedBox();
+        }
+      }
+
+      if (builders.containsKey(tag)) {
+        final Widget? builderChild =
+            builders[tag]!.visitElementAfterWithContext(
+          delegate.context,
+          element,
+          styleSheet.styles[tag],
+          _inlines.isNotEmpty ? _inlines.last.style : null,
         );
+        child = builderChild ?? defaultChild();
       } else {
-        child = const SizedBox();
+        child = defaultChild();
       }
```

Comment:
Optional: I think this would be simpler as just:

```dart
final MarkdownElementBuilder? builder = builders[tag];
Widget child = builder?.visitElementAfterWithContext(...) ?? defaultChild();
```

That avoids the duplication of saying that `defaultChild()` is the fallback, and also avoids the deferred initialization of `child`. (The counter-argument is that it's a pretty long ternary statement once you fill in the [...]; I know others have a lower threshold for ternaries than I do, so if you think it's too much that's fine.)

Reply:
Yes, that's simpler, and I don't mind the ternary length because it eliminates the whole conditional block.